support.sh

```diff
@@ -4,34 +4,70 @@
 DOCKER="${DOCKER:-docker}"
 NSENTER="${NSENTER:-nsenter}"
 BRIDGE="${BRIDGE:-bridge}"
-BRCTL="${BRCTL:-brctl}"
 IPTABLES="${IPTABLES:-iptables}"
+IPVSADM="${IPVSADM:-ipvsadm}"
+IP="${IP:-ip}"
 
 NSDIR=/var/run/docker/netns
-BRIDGEIF=br0
 
 function die {
     echo $*
     exit 1
 }
 
-type -P ${DOCKER} > /dev/null || die "This tool requires the docker binary"
-type -P ${NSENTER} > /dev/null || die "This tool requires nsenter"
-type -P ${BRIDGE} > /dev/null || die "This tool requires bridge"
-type -P ${BRCTL} > /dev/null || die "This tool requires brctl"
-type -P ${IPTABLES} > /dev/null || die "This tool requires iptables"
+function echo_and_run {
+  echo "#" "$@"
+  eval $(printf '%q ' "$@") < /dev/stdout 2>/dev/null
+}
+
+type -P ${DOCKER} > /dev/null || echo "This tool requires the docker binary"
+type -P ${NSENTER} > /dev/null || echo "This tool requires nsenter"
+type -P ${BRIDGE} > /dev/null || echo "This tool requires bridge"
+type -P ${IPTABLES} > /dev/null || echo "This tool requires iptables"
+type -P ${IPVSADM} > /dev/null || echo "This tool requires ipvsadm"
+type -P ${IP} > /dev/null || echo "This tool requires ip"
+
+echo "Host Configuration"
+echo_and_run ${IPTABLES} -w1 -n -v -L -t filter | grep -v '^$'
+echo_and_run ${IPTABLES} -w1 -n -v -L -t nat | grep -v '^$'
+echo_and_run ${IPTABLES} -w1 -n -v -L -t mangle | grep -v '^$'
+printf "\n"
 
-echo "iptables configuration"
-${IPTABLES} -w1 -n -v -L -t filter
-${IPTABLES} -w1 -n -v -L -t nat
-echo ""
+echo "Host addresses and routes"
+echo_and_run ${IP} -o -4 address show
+echo_and_run ${IP} -4 route show
+printf "\n"
 
 echo "Overlay network configuration"
-for networkID in $(${DOCKER} network ls --filter driver=overlay -q) ; do
-    echo "Network ${networkID}"
-    nspath=(${NSDIR}/*-$(echo ${networkID}| cut -c1-10))
-    ${DOCKER} network inspect -v ${networkID}
-    ${NSENTER} --net=${nspath[0]} ${BRIDGE} fdb show ${BRIDGEIF}
-    ${NSENTER} --net=${nspath[0]} ${BRCTL} showmacs ${BRIDGEIF}
-    echo ""
+for networkID in $(${DOCKER} network ls --filter driver=overlay -q) "ingress_sbox"; do
+    echo "nnn Network ${networkID}"
+    if [ "${networkID}" != "ingress_sbox" ]; then
+        nspath=(${NSDIR}/*-${networkID:0:10})
+        ${DOCKER} network inspect --verbose ${networkID}
+    else
+        nspath=(${NSDIR}/${networkID})
+    fi
+    echo_and_run ${NSENTER} --net=${nspath[0]} ${IP} -o -4 address show
+    echo_and_run ${NSENTER} --net=${nspath[0]} ${IP} -4 route show
+    echo_and_run ${NSENTER} --net=${nspath[0]} ${IP} -4 neigh show
+    echo_and_run ${NSENTER} --net=${nspath[0]} ${IPTABLES} -w1 -n -v -L -t filter | grep -v '^$'
+    echo_and_run ${NSENTER} --net=${nspath[0]} ${IPTABLES} -w1 -n -v -L -t nat | grep -v '^$'
+    echo_and_run ${NSENTER} --net=${nspath[0]} ${IPTABLES} -w1 -n -v -L -t mangle | grep -v '^$'
+    echo_and_run ${NSENTER} --net=${nspath[0]} ${IPVSADM} -l -n
```

Comment:
was it intentional to remove the `bridge fdb show`?
that will give an output like:
```
33:33:00:00:00:01 dev br0 self permanent
01:00:5e:00:00:01 dev br0 self permanent
fe:39:a0:4f:2b:db dev vxlan0 master br0 permanent
02:42:0a:00:11:1d dev vxlan0 dst 172.31.46.144 link-netnsid 0 self permanent
b6:dd:1c:32:33:db dev veth0 master br0 permanent
33:33:00:00:00:01 dev veth0 self permanent
01:00:5e:00:00:01 dev veth0 self permanent
```
and is useful to verify if the destination peer is correct

Reply:
No, that was not intentional - I'll go fix it